requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Versions are pinned to prevent pypi releases arbitrarily breaking
 # tests with new APIs/semantics. We want to update versions deliberately.
 ansible==2.2.2.0
+boto==2.45.0
```

Comment:
This is like an "optional" requirement, right?
I think we don't have a good way to express that.

What if we just install boto in both images, without adding it to `requirements.txt`?